route53_crossfeed_app.tf

```diff
@@ -271,57 +300,76 @@ resource "aws_route53_record" "crossfeed_staging_TXT" {
 }
 
-resource "aws_route53_record" "crossfeed_staging_api_AAAA" {
+resource "aws_route53_record" "crossfeed_staging_cd_api_AAAA" {
   provider = aws.route53resourcechange
 
   alias {
     name                   = "d2n7pvqicqw5zg.cloudfront.net."
     zone_id                = var.cloudfront_zone_id
     evaluate_target_health = false
   }
-  name    = "api.staging.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
+  name    = "api.staging-cd.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
   type    = "AAAA"
   zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
 }
 
-resource "aws_route53_record" "crossfeed_staging_api_CNAME1" {
+# The hosted_zone_id for the below records comes from https://docs.aws.amazon.com/general/latest/gr/elb.html
+# (ALBs in us-gov-west-1 region)
+resource "aws_route53_record" "crossfeed_staging_api_A" {
   provider = aws.route53resourcechange
 
-  name    = "_1cd918a94c59cbd2fdce9df0fdfc980f.api.staging.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
-  records = ["_f1fe0d0997fac6b9d221c8b482ac0ec2.jfrzftwwjs.acm-validations.aws."]
-  ttl     = 300
-  type    = "CNAME"
+  alias {
+    name                   = "crossfeed-stage-1792947306.us-gov-west-1.elb.amazonaws.com."
+    zone_id                = "Z33AYJ8TM3BH4J"
+    evaluate_target_health = false
+  }
+  name    = "api.staging.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
+  type    = "A"
   zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
 }
 
-resource "aws_route53_record" "crossfeed_staging_api_CNAME2" {
+resource "aws_route53_record" "crossfeed_staging_api_AAAA" {
   provider = aws.route53resourcechange
 
-  name    = "staging-cd.api.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
-  records = ["_73aacddb56ac20ef77492c962bf006d8.jfrzftwwjs.acm-validations.aws"]
+  alias {
+    name                   = "crossfeed-stage-1792947306.us-gov-west-1.elb.amazonaws.com."
+    zone_id                = "Z33AYJ8TM3BH4J"
+    evaluate_target_health = false
+  }
+  name    = "api.staging.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
+  type    = "AAAA"
+  zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
+}
+
+resource "aws_route53_record" "crossfeed_staging_api_CNAME1" {
+  provider = aws.route53resourcechange
+
+  name    = "_1cd918a94c59cbd2fdce9df0fdfc980f.api.staging.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
+  records = ["_f1fe0d0997fac6b9d221c8b482ac0ec2.jfrzftwwjs.acm-validations.aws."]
   ttl     = 300
   type    = "CNAME"
   zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
 }
 
-resource "aws_route53_record" "crossfeed_staging_api_CNAME3" {
+
+resource "aws_route53_record" "crossfeed_staging_api_CNAME2" {
   provider = aws.route53resourcechange
 
-  name    = "staging.api.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
-  records = ["crossfeed-stage-1792947306.us-gov-west-1.elb.amazonaws.com"]
+  name    = "_75f0c44adb3c8ec0023ddc11585227e8.api.staging-cd.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
```

Comment:
Moving this to a more appropriate location...

Just double-checking- is this meant to be:
`_75f0c44adb3c8ec0023ddc11585227e8.api.staging-cd.crossfeed...` or should it be
`_75f0c44adb3c8ec0023ddc11585227e8.api.staging.crossfeed...` (no "-cd")?

Confirming since the resource name here is `crossfeed_staging_api_CNAME2` (not `crossfeed_staging_cd_api_CNAME2`) and these things match in the case of the other records.

Reply:
It should have "-cd" in it. I can rename the resource name though to be consistent.

Reply:
@dav3r Updated the names now.

Reply:
Looks good, thanks!